src/data2img.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import scipy.misc
+import csv
+import os
+
+IMG_SIZE = 48
+csv_file_name = '../data/head.csv'
+data_output_dir = '../data/img'
+
+
+# based on https://stackoverflow.com/questions/312443/how-do-you-split-a-list-into-evenly-sized-chunks
+# based on https://stackoverflow.com/questions/434583/what-is-the-fastest-way-to-draw-an-image-from-discrete-pixel-values-in-python
+def csv2array(file_name):
+    img_list = []
+    csv_file = csv.DictReader(open(file_name))
+
+    for csv_row in csv_file:
+        data = np.zeros((IMG_SIZE, IMG_SIZE), dtype=np.uint8)
+        pixel_data = csv_row['pixels'].split()
+        for i in range(0, IMG_SIZE - 1):
```

Comment:
by the by - @utkuozbulak noticed that there are black lines at the bottom of the output, removing the `- 1` fixes it